contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -82,10 +82,13 @@ contract OETHHarvesterSimple is Initializable, Strategizable {
     }
 
     /// @notice Internal logic to harvest rewards from a strategy
-    function _harvestAndTransfer(address _strategy) internal {
+    function _harvestAndTransfer(address _strategy) internal virtual {
         // Ensure strategy is supported
         require(supportedStrategies[_strategy], "Strategy not supported");
 
+        // Store locally for some gas savings
+        address _strategist = strategistAddr;
```

Comment:
There seem to be only 1 call for the strategist address in the ternary expression below. We can delete this variable?

Reply:
That's inside a `for` loop :)

Reply:
ahh yes, I stand corrected :)

Reply:
Now that you point it out, I should move the `dripper` as well